### ISSUE TITLE
report_article throws error (after a while) with Octave 4

### ISSUE THREAD

**Issue by ex-ratt** [votchallenge/toolkit-legacy#119]
I get an error when calling `report_article` with Octave 4.0.0 on Linux Mint (based upon Ubuntu 16.04):
```
error: report_article: A(I) = X: X must have the same size as I
error: called from
    report_article at line 124 column 39
    run_analysis at line 19 column 1
```

My `run_analysis.m` looks like this:
```
addpath('/home/ex-ratt/Code/vot-toolkit'); toolkit_path;
[sequences, experiments] = workspace_load();
trackers = tracker_list('AT2', 'AT1', 'sKCF', 'KCF_SMXPC', 'KCF2014', 'ColorKCF');
context = create_report_context('report_vot2016_AT2');
report_article(context, experiments, trackers, sequences, 'spotlight', 'AT2');
```

Just ask if you need further information regarding this issue. It can be reproduced.

**Comment by lukacu** [comment]
As I have noted in version 5.0 release notes the analysis part of the toolkit does not work well at the moment, we are directing all our effort towards experiments running for VOT2017. We plan to restructure analysis after the challenge deadline. If you manage to determine the cause of the problem I will gladly review the solution, but I cannot promise that it will be fixed otherwise before end of June.

**Comment by ex-ratt** [comment]
Well, I should have read the release notes first. I am a newby when it comes to Matlab/Octave, but in order to get to the data, I have to find out what the function does anyway. Maybe I see something.

**Comment by lukacu** [comment]
Well, octave is a bit harder to use for debugging but I think I have already fixed similar bugs, it could be that something similar is fixed in report_challenge script which is pretty similar to report_article.

**Comment by bertinetto** [comment]
Any development on the issue?
Unfortunately we get the same error from run_analysis with the version 5.0.1 of the toolkit and Matlab.

```
In an assignment  A(I) = B, the number of elements in B and I must be the same.

Error in report_article (line 124)
    experiments_ranking_data(1:i:end) = scores(:, :, i);

Error in run_analysis (line 15)
```

**Comment by jvlmdr** [comment]
I tried to fix this problem in 8e5d281
